### ISSUE TITLE
PR: not able to export built in material parameters into standardised IFC psets

### ISSUE THREAD

**Issue by Moult** [Autodesk/revit-ifc#766]
### Problem Description

There are two ways material psets come out:

 1. https://github.com/Autodesk/revit-ifc/blob/b83595ff01f72f2a63242e00daf01508f2264287/Source/Revit.IFC.Export/Utility/MaterialPropertiesUtil.cs#L56-L58 from built-in parameters into hardcoded property sets called things like "Identity" or "Structural".
 2. https://github.com/Autodesk/revit-ifc/blob/b83595ff01f72f2a63242e00daf01508f2264287/Source/Revit.IFC.Export/Utility/MaterialPropertiesUtil.cs#L60 using a custom mapping from custom project / shared parameters where you can use the user defined psets txt config file to put it in whatever Pset you want.

The problem is that the first way does not put data into standard bSI psets. So the Revit built-in concrete compression data goes into a pset called "Structural", not "Pset_MaterialConcrete". And the second way cannot map these "built-in parameters" to my knowledge.

So my possible workarounds are to 1. patch the IFC to put it in the right spot and delete the other parameters that don't interest me / aren't validated or 2. create another project parameter and have the data duplicated in two spots.

### Revit Version

2024.0.x

### IFC for Revit Addon Version

24.x.x

### Windows Version

11 22H2

**Comment by lfcastel** [comment]
This is related to this issue: https://github.com/Autodesk/revit-ifc/issues/757

**Comment by Moult** [comment]
Yes, it's related but not exactly the same (#757 refers to attributes, whereas this is about properties). https://github.com/Autodesk/revit-ifc/issues/584 is also relevant and hints about this shortcoming.

**Comment by ArtemHnatko** [comment]
**Investigation Result:**
Current testing confirms mixed behavior regarding the export of Revit's built-in material parameters to standardized IFC Property Sets. This export occurs when the "Export material property sets" checkbox is enabled in the IFC export settings.

**Confirmed:** Built-in structural material properties (e.g., concrete compression data) are still being exported to non-standard IFC Property Sets (e.g., "Structural"), rather than their intended bSI counterparts (e.g., Pset_MaterialConcrete). This validates the primary concern outlined in the problem description.

**Mixed Behavior:** However, it's observed that certain other built-in material properties, such as thermal conductivity, are correctly placed within their respective standard bSI Psets (e.g., Pset_MaterialThermal). Other standard bSI Psets (e.g., Pset_MaterialCommon, Pset_MaterialMechanical) are also being utilized for some properties.

**Conclusion:** The bug is partially reproducible. While the core issue regarding the placement of structural properties into non-standard Psets persists, the exporter does appear to correctly map some built-in thermal and common material properties to standardized bSI Psets. This indicates a nuanced behavior rather than a complete failure to adhere to bSI standards for all built-in material parameters.

<img width="958" height="3320" alt="Image" src="https://github.com/user-attachments/assets/8f82a9aa-a807-4c34-96af-239d637ac6cc" />

**Comment by ArtemHnatko** [comment]
This issue is also reported in Jira as "REVIT-239670" and is currently in the "Open" status.
